Content.Shared/Chat/TypingIndicator/TypingIndicatorComponent.cs

```diff
@@ -14,7 +14,6 @@ public sealed partial class TypingIndicatorComponent : Component
     /// <summary>
     ///     Prototype id that store all visual info about typing indicator.
     /// </summary>
-    [ViewVariables(VVAccess.ReadWrite)]
     [DataField("proto", customTypeSerializer: typeof(PrototypeIdSerializer<TypingIndicatorPrototype>))]
-    public string Prototype = SharedTypingIndicatorSystem.InitialIndicatorId;
+    public string TypingIndicatorPrototype = "default";
```

Comment:
ProtoId

Reply:
Just to confirm: Your saying rename it to ProtoId? I was having trouble with what to name these things

Reply:
I mean change the field Type.

```suggestion
    public ProtoId<TypingIndicatorPrototype> TypingIndicatorPrototype = "default";
```